RFC-000-rfc-process-suggestions.md

```diff
@@ -0,0 +1,77 @@
+# RFC Zero - Open for suggestions to the RFC process

```

Comment:
add in a sample `yaml` block for front-matter, and we set some default values.  I think a discussion around what fields we have here could be a followup.  I'd propose we start with `proposalDate` and `status`.  Happy to add more now, but we could keep it simple for now.  

```
---
proposalDate: <yyyy-mm-dd>
status: <proposal|accepted> _we could add more versions_
---
```

> By adding the above `yaml`, it allows the RFC to be machine-readable by static-site generators and our doc-tooling.

Reply:
I've incorporated these suggestions into the RFC.